tus_storagehandler/consts.py

```diff
@@ -0,0 +1,22 @@
+"""This module handles the loading of environment variables for MinIO configuration.
+
+Environment variables expected:
+- MINIO_HOSTNAME: The hostname for the MinIO server.
+- MINIO_PORT: The port for the MinIO server.
+- MINIO_ACCESS_KEY: The access key for MinIO.
+- MINIO_SECRET_KEY: The secret key for MinIO.
+- MINIO_IS_SECURE: Boolean to determine if the connection is secure.
+- MINIO_BUCKET: The name of the bucket in MinIO.
+"""
+
+import os
+
+MINIO_HOSTNAME = os.getenv("MINIO_HOSTNAME")
+if MINIO_HOSTNAME is None:
+    raise ValueError("MINIO_HOSTNAME environment variable is not set")
+MINIO_PORT = os.getenv("MINIO_PORT")
+MINIO_ENDPOINT = f"{MINIO_HOSTNAME}:{MINIO_PORT}"
+MINIO_ACCESS_KEY = os.getenv("MINIO_ACCESS_KEY")
+MINIO_SECRET_KEY = os.getenv("MINIO_SECRET_KEY")
+MINIO_IS_SECURE = os.getenv("MINIO_IS_SECURE", "false").lower() in ["true"]
+MINIO_BUCKET = os.getenv("MINIO_BUCKET_NAME")
```

Comment:
I think none of these are constants. Rather, they are config variables that you want the user/admin to set when deploying the service. Possibly you may even want to set the bucket on the fly.

If you use FOCA, then you will get the config handling for free. You can do either of those two things:
- Just add the config params that you would like to define to `config.yaml` (maybe in a section `minio`) - and they will be available in the app config.
- BETTER: You can add them to a section `custom` (again, probably under a subsection `minio`) and provide to FOCA a module with Pydantic models for your config - FOCA will then validate the config against those models, making sure the user/admin provides valid inputs.

@JaeAeich has followed the second route for his project in this PR: https://github.com/elixir-cloud-aai/TESK/pull/185

If you want to use FOCA, it would probably make sense to first create a PR with the FOCA boilerplate - it should be pretty similar to @JaeAeich's PR above

Reply:
The FOCA boilerplate is already added right, so we will just need the 2nd part.

Reply:
Yes, of course - if it's already added, even better. Just make sure you expose these parameters as config variables in `config.yaml` so that the user/admin can set them declaratively, together with all the others in one document. And ideally do it in section `custom` - which is a special section for FOCA that it tries to validate if you pass it a module with models.

Reply:
Please re-request a review when you are done with these :pray: